packages/slonik/src/dbFilters.ts

```diff
@@ -73,16 +82,17 @@ const applyFiltersToQuery = (
 
   const applyFilters = (
     filters: FilterInput,
-    tableName: string,
+    tableIdentifier: IdentifierSqlToken,
     not = false
   ) => {
     if (filters.AND) {
-      for (const filterData of filters.AND) applyFilters(filterData, tableName);
+      for (const filterData of filters.AND)
+        applyFilters(filterData, tableIdentifier);
     } else if (filters.OR) {
       for (const filterData of filters.OR)
-        applyFilters(filterData, tableName, true);
+        applyFilters(filterData, tableIdentifier, true);
```

Comment:
Use the string `tableName` instead of a package-dependent `tableIdentifyer`. It's safer, way more legible to other developers.